test/local/senders/email.js

```diff
@@ -861,27 +861,40 @@ describe(
         })
 
         it(
-          'sends request to fxa-email-server when the email pattern is right',
+          'sends request to fxa-email-service when the email pattern is right',
           function() {
             const message = {
               email: 'emailservice.foo@restmail.net',
               subject: 'subject',
               template: 'verifyLoginEmail',
               uid: 'foo'
             }
+            mailer.sesConfigurationSet = 'wibble'
 
             return mailer.send(message)
               .then(
                 response => {
-                  assert(mailer.emailService.sendMail.called)
+                  assert(mailer.emailService.sendMail.calledOnce)
                   assert(! mailer.mailer.sendMail.called)
-                  assert.equal(mailer.emailService.sendMail.args[0][0].to, 'emailservice.foo@restmail.net')
-                  assert.equal(mailer.emailService.sendMail.args[0][0].subject, 'subject')
-                  assert.equal(mailer.emailService.sendMail.args[0][0].headers['X-Template-Name'], 'verifyLoginEmail')
-                  assert.equal(mailer.emailService.sendMail.args[0][0].headers['X-Email-Service'], 'fxa-email-service')
-                  assert.equal(mailer.emailService.sendMail.args[0][0].headers['X-Email-Sender'], 'ses')
-                  assert.equal(mailer.emailService.sendMail.args[0][0].headers['X-Uid'], 'foo')
-                  assert.equal(typeof mailer.emailService.sendMail.args[0][1], 'function')
+
+                  const args = mailer.emailService.sendMail.args[0]
+
+                  assert.equal(args.length, 2)
+                  assert.equal(args[0].to, 'emailservice.foo@restmail.net')
+                  assert.equal(args[0].subject, 'subject')
+
+                  const headers = args[0].headers
+
+                  assert.equal(headers['X-Template-Name'], 'verifyLoginEmail')
+                  assert.equal(headers['X-Email-Service'], 'fxa-email-service')
+                  assert.equal(headers['X-Email-Sender'], 'ses')
+                  assert.equal(headers['X-Uid'], 'foo')
```

Comment:
I aliased `args` and `headers` in this test to make them a bit more readable (also newlines are cool).